tokens/blocket.se/card.yml

```
@@ -0,0 +1,7 @@
+---
+token: maps
+
+shadow:
```

Did we remove `elevation` on purpose? Thinking of `elevation-shadow-card` token 🤔 But maybe you've decided about that somewhere
```suggestion
elevation:
  shadow:
```

---

Had a side discussion with @adidick yesterday and kind of decided that the token names in general are a bit lengthy... in this case "elevation" is somewhat extraneous information as there is no other types of elevations or shadows. Or what do you think? 🙂

---

Sounds like a decision that could've been mentioned in the PR description at least ;)